apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -0,0 +1,109 @@
+import React, {Component, PropTypes, Children} from 'react';
```

Comment:
Might be good to have a file level description of this component in here somewhere.